hack/bundle.sh

```diff
@@ -25,39 +25,81 @@ if [ ! -d "${DIR_OVERLAY}" ]; then
     exit 2
 fi
 
-echo "##############################################"
-echo "# id      : ${BUNDLE_ID}"
-echo "# version : ${ADDON_VERSION}"
-echo "# dir     : ${BUNDLE_DIR}"
-echo "# package : ${BUNDLE_NAME}"
-echo "# base    : ${BUNDLE_BASE}"
-echo "# channel : ${BUNDLE_CHANNEL}"
-echo "##############################################"
-
-kustomize build "${DIR_OVERLAY}" | operator-sdk generate bundle \
+function bundle_external_operators() {
+  local kustomization=$1
+  local container_image=''
+  local version=''
+
+  container_image=$(printf '%s\n' "$kustomization" | yq 'select(.kind=="Deployment").spec.template.spec.containers[0].image')
+  version=$(printf '%s' "$container_image" | cut -f 2 -d ':' | tr '[:upper:]' '[:lower:]')
+  version=${version:0:25}
+  container_image_name=$(printf '%s' "$container_image" | cut -f 1 -d ':')
+  container_image_sha256=$(skopeo inspect "docker://$container_image" --format "{{.Digest}}")
```

Comment:
Ideally this should happen in the cos-fleetshard pipeline [1][2] so installing the service with kustomize would produce (almost) the same result 

[1] https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/main/.github/workflows/build-tag.yaml#L75-L89
[2] https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/main/.github/workflows/build-main.yaml#L91-L102

Reply:
Ideally, yes. But we need the tag in this step to be able to extract the version name to OLM, which requires semantic versioning and is extracted from the tag name. Any ideas on how to overcome this for now?

Reply:
I think can grab such info from from an annotation/label. wdyt ?